config/checkstyle_checks.xml

```diff
@@ -55,6 +55,11 @@
     <property name="offCommentFormat" value="CSOFF\: .*"/>
     <property name="onCommentFormat" value="CSON\: .*"/>
   </module>
+  <module name="SuppressWithNearbyTextFilter">
+    <property name="nearbyTextFormat"
+              value="Consider using special escape sequence instead of octal.*Unicode"/>
+    <property name="idFormat" value="lineLengthXml"/>
```

Comment:
We probably inherited Format from old modules, but we do not have to do this.

Reply:
Done.